test/integration/etcd/data.go

```diff
@@ -168,14 +168,6 @@ func GetEtcdStorageDataForNamespace(namespace string) map[schema.GroupVersionRes
 		},
 		// --
 
```

Comment:
same here, revert this change, it's not required for integration tests, which aren't build-version aware

Reply:
> same here, revert this change, it's not required for integration tests, which aren't build-version aware

I think integration tests should become version aware (somehow) to ensure that we're testing what we actually want to enforce.  How we make that happen, I'm open to suggestions for.

Reply:
For this test specifically, which checks the persisted path and version of every served type, it should exercise every API the server is possibly capable of serving. I'd expect it to exercise these until the storage for these versions is actually removed.